src/renderer/components/top-nav/top-nav.scss

```diff
@@ -23,7 +23,7 @@
 
   @media only screen and (width >= 961px) {
```

Comment:
Thanks @4yman-el! Only feedback: It does look a bit odd having the surrounding spacing change. How about this?
```
media only screen and (width >= 1122px) {
        display: grid;
        grid-template-columns: 1fr 720px 1fr;
    }
```

![Screenshot_20240629_135756](https://github.com/FreeTubeApp/FreeTube/assets/84899178/d39bbe2c-32fe-4ed1-a2de-27f592eac967)

Reply:
It looks better. Thanks!

Reply:
Also update the width breakpoint to 1122px to prevent intermediate breakage. Sorry, I couldn't originally offer it as a multiline `suggestion` because I was in unified view.

Reply:
No problem!